.travis.yml

```diff
@@ -22,15 +22,45 @@ addons:
       - findutils
       - libeigen3-dev
 
+services:
+  - docker
+
 stages:
   # order stages
+  - name: interface64
   - name: precise
   - name: trusty
   - name: xenial
   - name: coverage
 
 jobs:
   include:
+  # xenial <=> interface64: "recent" systems with ICB
+  #- stage: interface64
+  #  dist: xenial
+  #  install: sudo apt-get install libmkl-gf-ilp64
+  #  script: dpkg -L libmkl-gf-ilp64
+  - stage: interface64
+    dist: xenial
+    script: |
+      sudo docker pull ubuntu                                                                                      \
+      &&                                                                                                           \
+      sudo docker run ubuntu /bin/sh -c                                                                            \
+      "cat /etc/os-release                                                                                      && \
+       apt-get -y update                                                                                        && \
+       apt-get -y install software-properties-common                                                            && \
+       apt-add-repository 'http://archive.ubuntu.com/ubuntu bionic         main restricted universe multiverse' && \
+       apt-add-repository 'http://archive.ubuntu.com/ubuntu bionic-updates main restricted universe multiverse' && \
```

Comment:
Running debian. If somebody has bionic installed, can you check/test which repo provides libmkl-gf-ilp64 ?
If we get this package, CI could test ILP64 support.